src/solver/conv_hip_implicit_gemm_fwd_v4r4_xdlops.cpp

```diff
@@ -236,7 +245,7 @@ void PerformanceImplicitGemmForwardV4R4Xdlops::EuristicInit(const ConvolutionCon
     });
 
     // second round: really valid
-    if(!tmp.IsReallyValid(ctx))
+    if(!(tmp.IsReallyValid(ctx) && tmp.IsFastToBeUsedForTuning(ctx)))
```

Comment:
If tmp is ```IsReallyValid``` but not ```IsFastToBeUsedForTuning```, we wouldn't want to loop over the tuning range to find another config that is ```IsReallyValid```, we could just accept tmp.

Reply:
ok, I see.